### ISSUE TITLE
Is OpenSouce Contribution to the evaluation dataset possible?

### ISSUE THREAD

**Issue by Kirushikesh** [patronus-ai/copyright-evals#1]
Hello Authors, Fascinated by the simplicity but an efficient solution. Currently if i am not wrong, the test data has 100 samples, which are used to test the LLM's. So is it possible for the opensource community to contribute more such prompts from the copyright content to the "test dataset". Allowing the opensource contribution adds diversity and quantity for a much generalised evaluation.

**Comment by sunitharavi9** [comment]
Yes, feel free to submit a PR with new prompts. We will add it to the collection
